client/asset/zec/zec.go

```diff
@@ -286,14 +298,253 @@ func connect(c rpcCaller) error {
 	if err := createAccount(0); err != nil {
 		return err
 	}
-	// When shielded pools are implemented, we'll use a separate, dedicated
-	// account that uses unified addresses with only and Orchard receiver type.
-	// if err := createAccount(1); err != nil {
-	// 	return err
-	// }
+	if err := createAccount(1); err != nil {
+		return err
+	}
 	return nil
 }
 
+func (w *zecWallet) lastShieldedAddress() (addr string, err error) {
+	if addrPtr := w.lastAddress.Load(); addrPtr != nil {
+		return addrPtr.(string), nil
+	}
+	accts, err := zListAccounts(w)
+	if err != nil {
+		return "", err
+	}
+	for _, acct := range accts {
+		if acct.Number != shieldedAcctNumber {
+			continue
+		}
+		if len(acct.Addresses) == 0 {
+			break // generate first address
+		}
+		lastAddr := acct.Addresses[len(acct.Addresses)-1].UnifiedAddr
+		w.lastAddress.Store(lastAddr)
+		return lastAddr, nil // Orchard = Unified for account 1
+	}
+	return w.NewShieldedAddress()
+}
+
+// ShieldedBalance list the last address and the balance in the shielded
+// account.
+func (w *zecWallet) ShieldedStatus() (status *asset.ShieldedStatus, err error) {
+	// z_listaccounts to get account 1 addresses
+	// DRAFT NOTE: It sucks that we need to list all accounts here. The zeroth
+	// account is our transparent addresses, and they'll all be included in the
+	// result. Should probably open a PR at zcash/zcash to add the ability to
+	// list a single account.
+	status = new(asset.ShieldedStatus)
+	status.LastAddress, err = w.lastShieldedAddress()
+	if err != nil {
+		return nil, err
+	}
+
+	status.Balance, err = zGetBalanceForAccount(w, shieldedAcctNumber)
+	if err != nil {
+		return nil, err
+	}
+
+	return status, nil
+}
+
+// Balance adds a sum of shielded pool balances to the transparent balance info.
+func (w *zecWallet) Balance() (*asset.Balance, error) {
+	bal, err := w.ExchangeWalletNoAuth.Balance()
```

Comment:
I would not object to a brief comment here touching on the "legacy" account concept you mention in https://github.com/decred/dcrdex/pull/2250#discussion_r1174427285, and the desire to aggregate transparent balance from any and all accounts, to match `listunspent`